ContextualPopupDecorator/ContextualPopup.js

```diff
@@ -0,0 +1,204 @@
+import EnactPropTypes from '@enact/core/internal/prop-types';
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import $L from '../internal/$L';
+import Button from '../Button';
+import Skinnable from '../Skinnable';
+
+import css from './ContextualPopup.module.less';
+
+/**
+ * An SVG arrow for {@link agate/ContextualPopupDecorator.ContextualPopup}.
+ *
+ * @class ContextualPopupArrow
+ * @memberof agate/ContextualPopupDecorator
+ * @ui
+ * @private
+ */
+const ContextualPopupArrow = kind({
+	name: 'ContextualPopupArrow',
+
+	propTypes: /** @lends agate/ContextualPopupDecorator.ContextualPopupArrow.prototype */ {
+		/**
+		 * Direction of ContextualPopupArrow.
+		 *
+		 * @type {('above'|'below'|'left'|'right')}
+		 * @default 'below'
+		 * @public
+		 */
+		direction: PropTypes.oneOf(['above', 'below', 'left', 'right'])
+	},
+
+	defaultProps: {
+		direction: 'below'
+	},
+
+	styles: {
+		css,
+		className: 'arrow'
+	},
+
+	computed: {
+		className: ({direction, styler}) => styler.append(direction, css.arrow)
+	},
+
+	render: (props) => (
+		<svg {...props} viewBox="0 0 30 30">
+			<path d="M15 2 L0 20 L30 20 Z" className={css.arrowFill} />
+		</svg>
+	)
+});
+
+const ContextualPopupRoot = Skinnable(
+	'div'
+);
```

Comment:
Could we make these code as one line `const ContextualPopupRoot = Skinnable('div')`?